examples/model_selection/plot_permutation_tests_for_classification.py

```diff
@@ -41,27 +41,28 @@
 # ----------------------
 #
 # Next, we calculate the
-# :func:`~sklearn.model_selection.permutation_test_score` using the original
-# iris dataset, which strongly predict the labels and
-# the randomly generated features and iris labels, which should have
-# no dependency between features and labels. We use the
+# :func:`~sklearn.model_selection.permutation_test_score` for both, the original
+# iris dataset (where features strongly predict the labels) and
+# the randomly generated features with iris labels (where no dependency between features
+# and labels is expected). We use the
 # :class:`~sklearn.svm.SVC` classifier and :ref:`accuracy_score` to evaluate
 # the model at each round.
 #
 # :func:`~sklearn.model_selection.permutation_test_score` generates a null
 # distribution by calculating the accuracy of the classifier
 # on 1000 different permutations of the dataset, where features
-# remain the same but labels undergo different permutations. This is the
+# remain the same but labels undergo different random permutations. This is the
 # distribution for the null hypothesis which states there is no dependency
 # between the features and labels. An empirical p-value is then calculated as
-# the percentage of permutations for which the score obtained is greater
-# that the score obtained using the original data.
+# the proportion of permutations for which the score obtained by the model trained on
```

Comment:
Substituting "percentage" by "proportion", because the range of possible values is from `1/(n_permutations + 1)` to `1.0`.